tests/libcxx/enc/enc.cpp

```diff
@@ -140,7 +140,7 @@ static int _pthread_join_hook(pthread_t enc_thread, void** value_ptr)
             enc_thread);
         oe_abort();
     }
-    int join_enc_key = it->first;
+    uint64_t join_enc_key = it->first;
```

Comment:
If you are changing enc_key from int to uint64_t, you may need to also change line 61,
int enc_key = 0; 
to
uint64_t enc_key = 0;

Reply:
Done. Was part of my next set of changes; but brought it in to this PR.